### PR TITLE
Hw rpi mqtt python br 2019.05

### PR DESCRIPTION
**Pull request by Sankalppund**
Buildroot Image with associated packages supporting for MQTT

### CODE REVIEW

File: base_external/configs/aesd_rpi_defconfig [hunk 1]
```diff
@@ -15,6 +19,27 @@ BR2_LINUX_KERNEL_DTS_SUPPORT=y
...
 BR2_PACKAGE_RPI_FIRMWARE=y
+BR2_PACKAGE_RPI_WIFI_FIRMWARE=y
+BR2_PACKAGE_PYTHON=y
+BR2_PACKAGE_PYTHON_HASHLIB=y
+BR2_PACKAGE_PYTHON_PAHO_MQTT=y
+BR2_PACKAGE_DROPBEAR=y
+BR2_PACKAGE_DROPBEAR_DISABLE_REVERSEDNS=y
+BR2_PACKAGE_MACCHANGER=y
+BR2_PACKAGE_MOSQUITTO=y
+BR2_PACKAGE_WIRELESS_TOOLS=y
```
Comment (dwalkes) [inline base_external/configs/aesd_rpi_defconfig]: @Sankalppund I don't think packages wireless/wpa are required, correct?  If so can you remove?

Comment (Sankalppund) [reply base_external/configs/aesd_rpi_defconfig]: I have added those packages to setup WiFi which will be required for MQTT communication.

Comment (dwalkes) [reply base_external/configs/aesd_rpi_defconfig]: I think you are missing some wifi support necessary content, so I'd rather just make this PR about support of mqtt via python.

**Review by Sankalppund — COMMENTED** [review]
I have removed the unnecessary packages and kept packages supporting MQTT.

File: base_external/configs/aesd_rpi_defconfig (hunk 1 above)
Comment (Sankalppund) [reply base_external/configs/aesd_rpi_defconfig]: I have made the necessary changes and now this PR is supporting MQTT via python.

**Pull request merged**